lib/simulator-xcode-6.js

```diff
@@ -610,6 +611,14 @@ class SimulatorXcode6 extends EventEmitter {
     `]);
   }
 
+  async dismissDatabaseAlert (increase = true) {
+    let button = increase ? 'Increase' : 'Cancel';
+    log.debug(`Attempting to dismiss database alert with '${button}' button`);
+    await exec('osascript', ['-e', `
+      activate application "Simulator"
      tell application "System Events"
      	tell process "Simulator"
      		click button "${button}" of window 1
      	end tell
      end tell
```

Comment:
this is kind of weird looking, are the `^M` necessary instead of `\\\n` or something similar?